evergreen.yml

```diff
@@ -73,8 +73,10 @@ functions:
           export STATIC_CODE_ANALYSIS_NAME="mongo-odbc-driver.sast.sarif"
           if [[ "${triggered_by_git_tag}" != "" ]]; then
             # tag should be formatted as `v<ODBC major>.<minor>.<patch>-libv<libmongosqltranslate major>.<minor>.<patch>`
-            export RELEASE_VERSION=$(echo ${triggered_by_git_tag} | cut -d'-' -f1 | sed s/v//)
-            export LIBMONGOSQLTRANSLATE_VERSION=$(echo ${triggered_by_git_tag} | cut -d'-' -f2 | sed s/libv//)
+            # we split on `-libv` to get the two versions
+            read RELEASE_VERSION LIBMONGOSQLTRANSLATE_VERSION <<< $(echo ${triggered_by_git_tag} | awk -F '-libv' '{print $1, $2}')
+            export RELEASE_VERSION="$RELEASE_VERSION"
+            export LIBMONGOSQLTRANSLATE_VERSION="$LIBMONGOSQLTRANSLATE_VERSION"
```

Comment:
A check for the valid format can be added, up to you if you think it will be helpful.  Something like:
```sh
    if [[ "${triggered_by_git_tag}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+-libv[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
        read RELEASE_VERSION LIBMONGOSQLTRANSLATE_VERSION <<< $(echo "${triggered_by_git_tag}" | awk -F '-libv' '{print $1, $2}')
        export RELEASE_VERSION
        export LIBMONGOSQLTRANSLATE_VERSION
    else
        echo "Error: Git tag is not correctly formatted: ${triggered_by_git_tag}"
        exit 1
    fi
```